src/convertToObject.js

```diff
@@ -3,16 +3,21 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map(item => item.split(':'))
+    .filter(item => item.length > 1)
+    .reduce((style, value) => {
+      style[value[0].trim()] = value[1].trim();
```

Comment:
We can improve this algorithm and make it just in 1 iteration, instead of 3. Let's do it.